ObjectState/TrackSends.cpp

```diff
@@ -432,3 +454,25 @@ void TrackSends::GetChunk(WDL_FastString* chunk)
 	for (int i = 0; i < m_sends.GetSize(); i++)
 		m_sends.Get(i)->GetChunk(chunk);
 }
+
+void TrackSends::DisplayStoreSendEnvError(MediaTrack* tr, const char* envName)

```

Comment:
Instead of taking a const char* and doing string comparison, maybe this could take a Enum (integer comparison)?